crates/re_time_panel/src/paint_ticks.rs

```diff
@@ -68,7 +76,8 @@ fn paint_time_range_ticks(
                 &ui.clip_rect(),
                 time_range, // ns
                 next_grid_tick_magnitude_ns,
-                |ns| Time::from_ns_since_epoch(ns).format_time_compact(),
+                // TODO(paris): Fix 'closures can only be coerced to `fn` types if they do not capture any variables' error.
+                |ns| Time::from_ns_since_epoch(ns).format_time_compact(false),
```

Comment:
the problem is with the signature of `paint_ticks`. Right now it takes the equivalent of a C function pointer:
`format_tick: fn(i64) -> String,`

Change it to `format_tick: impl Fn(i64) -> String` essentially making it a template method so it can take arbitrary objects that fullfil the `Fn(i64) -> String` contract (like passing a functor or a lambda in C++)

Reply:
I see great, that works, thank you.